### ISSUE TITLE
[Prot Paladin] Issue with scaling (Damage taken)

### ISSUE THREAD

**Issue by XsExcess** [simulationcraft/simc#2906]
Trying to redo my Stat weights for the epic tiers of gear and sort out some kind of trinket ranking, and I've noticed that I'm getting abysmally low Stamina scores. Tried TMI and Tank Dummy (allowing longer durations to get the Tank dummy to ramp) along with different tiers of "boss" testing to see if it was just trying to say I over geared the content it was Simming for, but even with Mythic tier on both they give ~0.1 (normalised), when the expected should be ~1500%+ of that as an absolute minimum, likely significantly more as I know for a fact my Stamina/Haste are too low to power through with skill alone.

Damage scaling in the graphs seems reasonable (at least for TMI), and trying a reforge plot shows that Stamina is dropping in relation to Strength (parry) as it should do, but changing it from a personal to pre-made (which is below my ilvl, but more balanced, with bonuses etc.) gives similar, if a lot more extreme, and very strange looking results (to be expected with set bonuses and trinkets in the mix).

This is coming from 01, in blue heroic 110 gear, to 02 and 03 (nightly) in average Mythic gear, all no legendaries, no procs, no nothing.
Best Comparison I can give, to put some values to my words (normalised strings):
Strength=1, Stamina=2.83, CritRating=0.92, HasteRating=1.43, MasteryRating=1.09, Versatility=1.02 (Blue 110, T19H Sim I think)
Strength=1, Stamina=0.01, CritRating=1.57, HasteRating=1.20, MasteryRating=2.13, Versatility=1.87 (Solid Epic, T19M Sim)
Strength=1, Stamina=-0.09, CritRating=1.43, HasteRating=1.29, MasteryRating=2.67, Versatility=1.80 (Premade T19M Sim, minus stamina, Impressive)

My initial reason for thinking something was up was when I managed to Sim a negative rating on my personal gear for Stam, which while technically possible given Paladin's interactions, seems unlikely to be plausible.
http://i.imgur.com/s57BQ7v.png Standard settings, with obvious tweaks depending on the changes described, like doubling the time etc. for Tank Dummy and some initial tests with Lag/Skill to check the interaction was there (was).

Also checked with Theck initially to ensure TMI wasn't broken by 02/03, as I know hes a lot less active these days, before/while attempting to remove the TMI algorithm from the equation. He confirmed it looked like something other than TMI.

Edit: Oh and I tried to get back to 01, but wasn't able to easily go back to it, it was still doing some 02+ looking things during the install process and ended up spitting out the same sort of results.

**Comment by scamille** [comment]
One simple thing to debug scale factors is to do them manually:
Run 1 base sim, and then another delta-sim with the stamina (eg. +500) added manually. Add maximize_reporting=1 to the override tab, to get even more info in your report.
Then you can just manually compare TMI and all the other goods (stamina, hp, hp-timeline, resource statistics, etc. ) between the two reports.

That might allow you to pinpoint the problem to something more precise, eg. why stamina/hp values are off, etc.

**Comment by XsExcess** [comment]
Tried everything manually, using 50k to ensure as much accuracy as I could within reason, and I've got (alomost) the results I'd expect to get (ie from 01 release, the Blue set in the OP).
![Results](http://i.imgur.com/gwkkU5x.png?1)

This was all done using manual stats (the ones due to be changed), with +500 (or 750 for the Stam x1.5 result) added, with a few reruns on the Strength result to ensure everything was spitting out right, not sure why it was so low but its the reason I didn't try to display them as Normalized weights like the OP.

Also, here is the +500 all simcrafted result, showing very similar results to the premade, although this would now be similar to around ilvl 900 (With ~10% more armor).
[results_html.zip](https://github.com/simulationcraft/simc/files/504953/results_html.zip)

While this also shows that something strange is happening to effectively reverse the output of what we'd expect, both at normal levels and significantly higher ones, I also looked more closely into Mythic tanking requirements, and found that they more closely resemble ~700k DTPS, where as the current tuning has them hovering around 500-550k DTPS in SimC. This is probably also a separate, all be it less important, issue to be addressed. Added some extra Sims with the other preset bosses and other tiers of character to confirm its not the bosses causing upside down scales, and I was unable to figure out from the more detailed Sim information command anything that could point me to anywhere specifically causing it (mostly because I'm unable to create a normal Sim for comparison).
